src/hooks/sdk/useSelectVideoInputDevice.tsx

```diff
@@ -6,7 +6,9 @@ import { useCallback } from 'react';
 import { useLocalVideo } from '../../providers/LocalVideoProvider';
 import { useMeetingManager } from '../../providers/MeetingProvider';
 
-export const useSelectVideoInputDevice = () => {
+export const useSelectVideoInputDevice = (): ((
+  deviceId: string
```

Comment:
This should now be supporting both `Device` and `VideoTransformDevice` as well as `null` right?

Reply:
No, `null` is no longer supported by `selectVideoInputDevice` in V3. The equivalent will be a new API `unselectVideoInputDevice()`.

Reply:
The deviceId type change will be in next PR. the type will be `Device | VideoTransformDevice` (`VideoInputDevice`)

Reply:
I think `unselectVideoInputDevice` would not be good as it seems from JS SDK V3 we just start stop the video input device and allow choosing only a valid input device.